### DISCUSSION TITLE
Generate 2 css files from 1

### DISCUSSION THREAD

**Discussion by Nevoss** [tailwindlabs/tailwindcss#2039] [Help]
Hey, 
I am thinking about taking the RTL approach to another level,
to make my files very slim I am trying to create new classes like: 'ms' - margin start, and 'me' - margin end.
and then it will generate 2 files, one for LTR and one for RTL.
Does someone have experience with that? generating 2 files from 1 CSS file?

Thank you for any help. :smile:

**Comment by 20lives** [comment]
What's wrong with the existing solution offered in [tailwindcss-rtl](https://github.com/20lives/tailwindcss-rtl) ?
